kasa/effects.py

```diff
@@ -0,0 +1,296 @@
+"""Module for light strip effects (LB*, KL*, KB*)."""
+
+from typing import List, cast
+
+EFFECT_AURORA = {
+    "custom": 0,
+    "id": "xqUxDhbAhNLqulcuRMyPBmVGyTOyEMEu",
+    "brightness": 100,
+    "name": "Aurora",
+    "segments": [0],
+    "expansion_strategy": 1,
+    "enable": 1,
+    "type": "sequence",
+    "duration": 0,
+    "transition": 1500,
+    "direction": 4,
+    "spread": 7,
+    "repeat_times": 0,
+    "sequence": [[120, 100, 100], [240, 100, 100], [260, 100, 100], [280, 100, 100]],
+}
+EFFECT_BUBBLING_CAULDRON = {
+    "custom": 0,
+    "id": "tIwTRQBqJpeNKbrtBMFCgkdPTbAQGfRP",
+    "brightness": 100,
+    "name": "Bubbling Cauldron",
+    "segments": [0],
+    "expansion_strategy": 1,
+    "enable": 1,
+    "type": "random",
+    "hue_range": [100, 270],
+    "saturation_range": [80, 100],
+    "brightness_range": [50, 100],
+    "duration": 0,
+    "transition": 200,
+    "init_states": [[270, 100, 100]],
+    "fadeoff": 1000,
+    "random_seed": 24,
+    "backgrounds": [[270, 40, 50]],
+}
```

Comment:
Would it make sense to convert these to dataclasses? That would also allow sub-dataclasses to document the embedded structures.

Reply:
I expected that users will want to pass in custom ones, and the api is expecting a `dict` already so I'm not sure its worth converting is back and forth.

Reply:
Fair enough, I was just wondering if having them as containers would make it more palatable by explicitly documenting the items. It's fine as it is for me.